docs/UserGuide.md

```diff
@@ -54,6 +54,31 @@ If you can type fast, TAB can get your Lab management tasks done faster than tra
     - [Edit labs](#editing-individual-labs--labedit)
     - [Remove labs](#removing-a-lab-labrm)
 
+### Things to note
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `…` after them can be used multiple times including zero times.<br>
+  e.g. `[t/TAG]…` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
```

Comment:
Perhaps we should change the pre-existing tag examples because it's a bit weird to tag students as `t/friend` and `t/family`

Reply:
HAHA yea we should.